### ISSUE TITLE
How to run custom resolvers along with neo4j-graphql-java library?

### ISSUE THREAD

**Issue by Anunayth** [neo4j-graphql/neo4j-graphql-java#82]
Is it possible to have my custom resolvers (using graphql-java) integrated and work in parallel with neo4j-graphql-java library? I want to use neo4j-graphql-java library for resolving queries for all objects, however for few of them i want to write my own resolvers.   

The [FAQ on github](https://github.com/neo4j-graphql/neo4j-graphql-java/#how-does-this-related-to-graphql-java) section says "If you wanted, you could combine graphql-java resolvers with this library to have a part of your schema handled by Neo4j"

From this statement i am assuming it can be done. If it is possible, please help me out on how can we do it?

**Comment by jexp** [comment]
Yes I think @Andy2003 had an example, you're right we should put it into an example file.

**Comment by Andy2003** [comment]
duplicate of #59
